app/actions/remote/preference.ts

```diff
@@ -48,23 +51,56 @@ export const saveFavoriteChannel = async (serverUrl: string, channelId: string,
         return {error: `${serverUrl} database not found`};
     }
 
-    let client;
     try {
-        client = NetworkManager.getClient(serverUrl);
+        // Todo: @shaz I think you'll need to add the category handler here so that the channel is added/removed from the favorites category
```

Comment:
Mention is nice, ticket is even better ;)

Reply:
@shazm can you create a ticket and handle this?